python/BioSimSpace/Sandpit/Exscientia/Protocol/_free_energy_mixin.py

```diff
@@ -96,35 +98,35 @@ def getPerturbationType(self):
         return self._perturbation_type
 
     def setPerturbationType(self, perturbation_type):
-        """
-        Set the perturbation type.
-
-        Parameters
-        ----------
-
-        perturbation_type : str
-            The type of perturbation to perform. Options are:
-             "full" : A full perturbation of all terms (default option).
-             "discharge_soft" : Perturb all discharging soft atom charge terms (i.e. value->0.0).
-             "vanish_soft" : Perturb all vanishing soft atom LJ terms (i.e. value->0.0).
-             "flip" : Perturb all hard atom terms as well as bonds/angles.
-             "grow_soft" : Perturb all growing soft atom LJ terms (i.e. 0.0->value).
-             "charge_soft" : Perturb all charging soft atom LJ terms (i.e. 0.0->value).
+        """Set the perturbation type.
+
+           Parameters
+           ----------
+
+           perturbation_type : str
+               The type of perturbation to perform. Options are:
+                "full" : A full perturbation of all terms (default option).
+                "discharge_soft" : Perturb all discharging soft atom charge terms (i.e. value->0.0).
+                "vanish_soft" : Perturb all vanishing soft atom LJ terms (i.e. value->0.0).
+                "flip" : Perturb all hard atom terms as well as bonds/angles.
+                "grow_soft" : Perturb all growing soft atom LJ terms (i.e. 0.0->value).
+                "charge_soft" : Perturb all charging soft atom LJ terms (i.e. 0.0->value).
+                "restraint" : Perturb the receptor-ligand restraint strength by linearly 
+                            scaling the force constants (0.0->value).
```

Comment:
Can this be changed so that the docstring starts on its own line and the subsequent documentation is aligned with the opening parenthesis, i.e. as in the original.